lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -90,9 +90,15 @@ our @ISA = qw/Text::Markdown/;
 sub _DoAutoLinks {
     my ($self, $text) = @_;
 
+    # auto-replace every http(s) reference which is not already either html
+    # 'a href...' or markdown link '[link](url)'
+    $text =~ s{(?<!['"(])(http[s]?://[^\s]*)}{<a href="$1">$1</a>}gi;
```

Comment:
Looking at the _DoAutoLinks code, it does not work on reference but its pass by value. It then returns the augmented string. So we should do `$text = $self->SUPER::_DoAutoLinks($text);`